cvat/apps/documentation/user_guide.md

```diff
@@ -359,406 +576,523 @@ Go to specified frame. Press ``~`` to highlight element.
 ![](static/documentation/images/image060.jpg)
 
 ---
-__Open Menu__ button
+**Open Menu** button
 
-It is the main menu for the annotation tool. It can be used to download, upload and remove annotations.
+It is the main menu of the annotation tool. It can be used to download, upload and remove annotation.
 
 ![](static/documentation/images/image051.jpg)
 
-As well it shows statistics about the current task, such as:
+Button assignment:
+
+- ``Open Task`` — open task in cvat dashboard
+- ``RunReID Merge`` — The ReID application uses deep learning model to perform
+an automatic bbox merging between neighbor frames. You can use "Merge" and
+"Split" functionality to edit automatically generated annotation.
+[Read more](/cvat/apps/reid)
+- ``Dump Annotation`` — download annotations from the task
+- ``Upload Annotation`` — uploading annotations to the task
+- ``Remove Annotation`` — remove annotations from current task
+- ``Settings`` — open the settings menu
+- ``Fullscreen Player`` — fullscreen player mode
+- ``Switch AAM`` — switch to attribute annotation mode
+- ``Help`` — open the shortkeys
+- ``Save Work`` — save annotations for the current job. The button has an indication of the saving process
+
+It also shows statistics about the current task, for example:
 - task name
 - type of performance on the task: ``annotation``, ``validation`` or ``completed task``
 - technical information about task
-- number of created bounding boxes, sorted by labels (e.g. vehicle, person) and type of annotation (polygons, boxes, etc.)
+- number of created bounding boxes, sorted by labels (e.g. vehicle, person) and
+type of annotation (polygons, boxes, etc.)
 
 ---
-__Filter__ input box
+**Filter** input box
 
 The way how to use filters is described in the advanced guide (below).
 
 ![](static/documentation/images/image059.jpg)
 
 ---
-__History / Undo-redo panel__
+**History / Undo-redo** panel
 
 Use ``Ctrl+Z`` for undo actions and  ``Ctrl+Shift+Z`` or ``Ctrl+Y`` to redo them.
 
 ![](static/documentation/images/image061.jpg)
 
 ---
-__Fill Opacity slider__
+**Propagation** input box
+
+Allows you to select how many frames of the selected object will be copied clicking ``Ctrl+B`` (50 by default)
+
+![](static/documentation/images/image101.jpg)
+
+---
+**Rotation** of an image
+
+Buttons are used to rotate the image clockwise / counterclockwise or use ``Ctrl+R``/``Ctrl+Shift+R``
+
+![](static/documentation/images/image102.jpg)
+
+---
+**Fill Opacity** slider
 
 Change opacity of every bounding box in the annotation.
 
 ![](static/documentation/images/image086.jpg)
 
-Opacity can be chaged from 0% to 100% and by random colors or white. If any white option is chosen, ``Color By`` scheme won't work.
+Opacity can be chaged from 0% to 100% and by random colors or white. If any
```

Comment:
I don't understand the sentence. Need to rephrase it:

and by random colors or white. If any white option is chosen, Color By scheme won't work.